homeassistant/components/shelly/binary_sensor.py

```diff
@@ -73,6 +73,11 @@
         default_enabled=False,
         removal_condition=is_momentary_input,
     ),
+    ("sensor", "extInput"): BlockAttributeDescription(
+        name="External input",
```

Comment:
```suggestion
        name="External Input",
```

Reply:
Other sensors are named with capital first letter on both words (e.g. `Power Factor`)

Reply:
Some are and some not ( e.g. `Firmware update` )

I would prefer lowercase as it's a compound description.

Reply:
> Some are and some not ( e.g. `Firmware update` )

So we also need to fix this sensor. @chemelli74 We need to be consistent with all sensors, for now only sensors which you added are with lowercase.